.storybook/preview.js

```diff
@@ -1,4 +1,4 @@
-import '../webapp/sass/profile.scss';
+// import '../webapp/sass/profile.scss';
```

Comment:
this is good for testing, but this will mess up all other storybook pages (since they assume some global css)